docs/advanced-topics.adoc

```diff
@@ -0,0 +1,41 @@
+ifdef::env-github,env-browser[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:outfilesuffix: .adoc
+endif::[]
+
+== Advanced Topics
```

Comment:
This is so great addition!

![dancing-bel-air](https://user-images.githubusercontent.com/719616/35748381-275d2a48-084e-11e8-8e87-275f2a55b72b.gif)